src/datasets/glam.md

```diff
@@ -0,0 +1,115 @@
+# GLAM datasets
+
+## Introduction
+
+The GLAM aggregation tables are useful for getting the SQL view of the data that drives GLAM if more exploration is required. GLAM aims to answer a majority of the "easy" questions of how a probe or metric has changed over time. Exploring the GLAM tables could take that a little farther. If you need to dive deeper or aggregate on a field that isn't included here, consider reading [todo].
+
+The GLAM tables:
+* Are aggregated at the client level, not the submission ping level
+* Provide a set of dimensions for subsets: channel, os, process or ping type
+* Are aggregated by build ID and version
+* For each aggregation, the distribution and percentiles over time are calculated
+
+### Firefox Desktop
+
+* Optional info?
+    * Pre-processed from Main telemetry to intermediate data with one row per client per metric per day, then aggregated normalizing across clients.
+    * Clients daily aggregates analagous to clients daily with:
+        * all metrics aggregated
+        * each scalar includes min, max, average, sum, and count aggregations
+        * each histogram aggregated over all client data per day
+        * each date is further aggregated over the dimensions: channel, os, version, build ID
+
+
+* Data source table
+    * `moz-fx-data-shared-prod.telemetry.client_probe_counts`
+* Data reference
+    * `os`: One of Windows, Mac, Linux, or NULL for all OSes
+    * `app_version`: Integer representing the major version
+    * `app_build_id`: The full build ID, or NULL if aggregated by major version
+    * `channel`: One of nightly, beta, or release
+    * `metric`: The name of the metric
+    * `metric_type`: The type of metric, e.g. "histogram-enumerated"
+    * `key`: The key if the metric is a keyed metric
+    * `process`: The process
+    * `client_agg_type`: The type of client aggregation used, e.g. "summed_histogram"
+    * `agg_type`: One of histogram or percentiles representing what data will be in the `aggregates` column
+    * `total_users`: The number of users that submitted data for the combination of dimensions
+    * `aggregates`: The data as a key/value record, either percentiles or histogram
+
+#### Sample query
+```sql
+SELECT
+  os,
+  app_version,
+  app_build_id,
+  channel,
+  metric,
+  metric_type,
+  key,
+  process,
+  client_agg_type,
+  agg_type,
+  total_users,
+  mozfun.glam.histogram_cast_json(aggregates) AS aggregates
+FROM
+  `moz-fx-data-shared-prod.telemetry.client_probe_counts`
+WHERE
+  metric="checkerboard_severity"
+  AND channel="nightly"
+  AND os IS NULL
+  AND process="parent"
+  AND app_build_id IS NULL
+```
+
+NOTE:
+* To query all OSes, use: `WHERE os IS NULL`
+* To query by build ID, use: `WHERE app_build_id IS NOT NULL`
+* To query by version, use: `WHERE app_build_id IS NULL`
+
+### Firefox for Android
```

Comment:
I'm hoping to autogenerate docs for tables that derive from a Glean Application in the Glean Dictionary soon (some of what's here will be very useful for that!)